docs/en/Getting-Started-on-Cluster.md

```diff
@@ -0,0 +1,348 @@
+the Alluxio journal and the worker storage directory in preparation for the master and worker to
+start.
+
+Please make sure this command is run on all the nodes.
+
+```bash
+$ ./bin/alluxio format
+```
+
+Now, we can start Alluxio! In this doc, we will start one Alluxio Master and two Alluxio Workers.
+
+On one of the node (naming it Master Node), run the following command to start Alluxio Master:
+
+```bash
+$ ./bin/alluxio-start.sh master
+```
+
+Congratulations! Alluxio master is now up and running! You can visit
+[http://ALLUXIO_MASTER_HOSTNAME:19999](http://ALLUXIO_MASTER_HOSTNAME:19999) to see the status of the Alluxio master.
+
+On the other two nodes (naming them Worker1 and Worker2, respectively), run the following command to start
+ Alluxio worker:
+
+```bash
+$ ./bin/alluxio-start.sh worker
+```
+
+In a few seconds, Alluxio workers will register with the Alluxio master.
+You can visit [http://ALLUXIO_WORKER1_HOSTNAME:30000](http://ALLUXIO_WORKER1_HOSTNAME:30000)
+to see the status of the Alluxio worker.
+
+## Use the Alluxio Shell
+
+Now that Alluxio is running, we can examine the Alluxio file system with the
+[Alluxio shell](Command-Line-Interface.html). The Alluxio shell enables many command-line operations
+for interacting with Alluxio. You can ssh to the Alluxio Master node, and invoke the Alluxio shell with
+the following command:
+
+```bash
+$ ./bin/alluxio fs
+```
+
+This will print out the available Alluxio command-line operations.
+
+For example, you can list files in Alluxio with the `ls` command. To list all files in the root directory, use the following command:
+
+```bash
+$ ./bin/alluxio fs ls /
+```
+
+Unfortunately, we do not have any files in Alluxio. We can solve that by copying a file into
+Alluxio. The `copyFromLocal` shell command is used to copy a local file into Alluxio.
+
+```bash
+$ ./bin/alluxio fs copyFromLocal LICENSE /LICENSE
+Copied LICENSE to /LICENSE
+```
+
+After copying the `LICENSE` file, we should be able to see it in Alluxio. List the files in
+Alluxio with the command:
+
+```bash
+$ ./bin/alluxio fs ls /
+drwxr-xr-x   ubuntu    ubuntu    26.22KB   09-22-2017 09:30:08:781  In Memory      /LICENSE
+```
+
+The output shows the file that exists in Alluxio, as well as some other useful information, like the
+size of the file, the date it was created, and the in-Alluxio status of the file.
+
+You can also view the contents of the file through the Alluxio shell. The `cat` command will print
+the contents of the file.
+
+```bash
+$ ./bin/alluxio fs cat /LICENSE
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+...
+```
+
+With this UFS configuration, Alluxio uses the specified S3 bucket as its UnderFileSystem (UFS).
+We can check whether this file exists in the S3 bucket.
```

Comment:
Combine with next paragraph

Reply:
I meant to combine "We can check whether this file exists in the S3 bucket." - the sentence is awkward as its own paragraph because it isn't true, and requires the next paragraph to make sense.

Reply:
done